doc/jointcal.tex

```diff
@@ -29,527 +43,603 @@
-hides two details implemented in the code: accounting for proper
-motions and differential effects of atmospheric refraction.
+
+The expression \ref{eq:astrometry-chi2} above depends on two sets
+of parameters: the parameters defining the mappings and the positions
+$F_{k}$. This expression hides two details implemented in the code:
+accounting for proper motions and differential effects of atmospheric
+refraction.
 
 Proper motions can be accounted for to predict the expected positions
-of objects and even be considered as fit parameters. At the moment we
-neither have code to detect that some (presumably stellar) object is
-moving, nor code to ingest proper motions from some external catalog.
+of objects and even be considered as fit parameters. At the moment
+we neither have code to detect that some (presumably stellar) object
+is moving, nor code to ingest proper motions from some external catalog.
 Each \ClName{FittedStar} has a flag that says whether it is affected
 by a proper motion and the proper motion parameters can all be fitted
 or not (see \S \ref{sec:indices_whattofit}).
 
-The code allows to account for differential chromatic effects of 
-atmospheric refraction, i.e. the fact that objects positions in the image
-plane are shifted by atmospheric refraction in a way that depends on 
-their color. The shift reads:
+The code allows to account for differential chromatic effects of atmospheric
+refraction, i.e. the fact that objects positions in the image plane
+are shifted by atmospheric refraction in a way that depends on their
+color. The shift reads: 
 \begin{equation}
-\delta X = k_b(c-c_0)\hat{n} \label{eq:refrac_corr}
+\delta S=k_{b}(c-c_{0})\hat{n}\label{eq:refrac_corr}
 \end{equation}
-where $k_b$ is a fit parameter (one per band $b$), $c$ is the color of
-the object in hand, $c_0$ is the average color, and $\hat{n}$ is the
-direction of the displacement in the tangent plane (i.e. a normalized
-vector along the parallactic direction, computed once for all for each
-Calexp).  We have not accounted for pressure variations because they
-are usually small, but it would not be difficult. The code accounts
-for color-driven differential effects within a given band, but ignores
-the differences across bands, would one attempt to fit images from
-different bands at the same time. Differences in recorded positions
-across bands will be accounted for in the fitted mappings. It is
-important to do so because we are fitting WCS's, and we want the fitted
-mappings to reflect at best the effects affecting measured
-positions. Since the color correction \ref{eq:refrac_corr} is not
-accounted for when using WCS's to transform measured position, we have
-made this correction zero on average. As for proper motions, fitting
-or not these refraction-induced differential position shifts is left
-to the user (see \S \ref{sec:indices_whattofit}).
+where $k_{b}$ is a fit parameter (one per band $b$), $c$ is the
```

Comment:
is the color derived from F_i, or something else?

Reply:
I don't know: this component of the code is not currently active, and I haven't had time to explore it yet.

I've decided to move sections 3.3 (The astrometric distortion model), 3.4 (Choice of projectors), 3.5 (Proper motions and atmospheric refraction) to after 3.7 (Photometry example), since they're not really necessary for explaining the general algorithm.